src/app.cr

```diff
@@ -103,6 +103,7 @@ spawn(same_thread: true) do
   begin
     PlaceOS::FrontendLoader::Loader.instance.start
   rescue error
+    puts error.inspect_with_backtrace
     PlaceOS::FrontendLoader::Loader::Log.error(exception: error) { "startup failed" }
```

Comment:
This `puts` is definitely redundant to the line below.


```suggestion
    PlaceOS::FrontendLoader::Loader::Log.error(exception: error) { "startup failed: #{error.message}" }
```

Reply:
the `Log` message isn't being output as the process exits before the async logs are flushed

After testing on with this image and the puts:

```
Listening on http://0.0.0.0:3000
level=[W] time=2022-04-27T03:32:07Z program=frontend-loader source=place_os.encryption message="using insecure default PLACE_SERVER_SECRET"
Hostname lookup for github.com failed: No address found (Socket::Addrinfo::Error)
  from /usr/share/crystal/src/socket/addrinfo.cr:176:9 in 'new'
  from /usr/share/crystal/src/http/client.cr:789:10 in 'io'
  from /usr/share/crystal/src/http/client.cr:671:5 in 'send_request'
  from /usr/share/crystal/src/http/client.cr:651:20 in 'execute'
  from /build/src/placeos-frontend-loader/api/remote/remote.cr:207:7 in 'download_archive'
  from /build/src/placeos-frontend-loader/api/remote/github.cr:71:13 in 'create_base_www'
  from /usr/share/crystal/src/gc/boehm.cr:110:5 in '->'
  from /usr/share/crystal/src/primitives.cr:266:3 in 'run'
  from ???
frontend-loader leaps through the veldt
```

we need to be using https://github.com/spider-gazelle/connect-proxy
or extending the built in HTTP::Client to use the proxy